rdflib/plugins/parsers/RDFVOC.py

```diff
@@ -0,0 +1,19 @@
+from rdflib.namespace.RDF import RDF
+from rdflib.term import URIRef
+
+
+class RDFVOC(RDF):
```

Comment:
Yes - if you feel deeply about having your software die (vs. warn), you can set RDF._fail to True.  If you feel deeply that *everyone* needs/wants to endure this behavior, we can set RDF._fail to True as the default and we can override it if needs be.

Reply:
Actually, the RDFVOC is the extended RDF that is used in the RDF XML parser.  I had to make it behave like a ClosedNamespace because the parser actually depended (well, the TEST cases depended) on ClosedNamespace behavior.  Was easier to do this way than try to blend Defined and Closed namespaces.

At some point we definitely need to do a bit 'o performance testing on these to see whether I've injected any unexpected slowness...

Reply:
re: failure for everyone. I don't have enough data to say what a sane default should be in the ideal case, but changing the existing default behavior is something that should be carefully considered, and definitely documented in the change log. That said, if the decision is to make the change so that all namespaces have consistent default behavior (which I think is a good idea), then 6.0.0 seems like an ideal time to do it.

Reply:
I think we have the first consequence of this just here: #1374